pkg/etcdenvvar/etcd_env.go

```diff
@@ -241,6 +242,22 @@ func getHardwareSpeedValues(envVarContext envVarContext) (map[string]string, err
 	return hwspeedhelpers.HardwareSpeedToEnvMap(speed)
 }
 
+func getEtcdDBSize(envVarContext envVarContext) (map[string]string, error) {
+	etcd, err := envVarContext.etcdLister.Get("cluster")
+	if err != nil {
+		return nil, fmt.Errorf("failed to retrieve etcd CR: %v", err)
+	}
+
+	if etcd.Spec.EtcdDBSize == nil {
```

Comment:
I would suggest not failing if this isn't set and just use the current default value. This will impact upgrades where the field doesn't exist

Reply:
👍🏽

Reply:
fixed 👍🏽  ptal